doc/guides/bluetooth/bluetooth-arch.rst

```diff
@@ -110,7 +110,7 @@ BLE-enabled builds that can be produced from the Zephyr project codebase:
   * :option:`CONFIG_BT_HCI` ``=y``
   * :option:`CONFIG_BT_HCI_RAW` ``=y``
   * :option:`CONFIG_BT_CTLR` ``=y``
-  * :option:`CONFIG_BT_LL_SW_SPLIT` ``=y`` (if using the open source Link Layer)
+  * :option:`CONFIG_BT_LL_SW` ``=y`` (if using the open source Link Layer)
```

Comment:
Are you sure you want to do this change?
This is likely to exist in many prj.conf in existing applications and in our downstream repositories. You are creating a big clean-up job for yourself.

@rugeGerritsen Any opinions on maintenance for this rename?

Reply:
If there no important reason to do the change, I would leave it as is. I did a similar change renaming the nRF BLE Controller to SoftDevice controller. It required updates in multiple repos. 

If you decide to rename the configuration, you should add least keep the old one for deprecation purposes

Reply:
I understand the arguments here, and will not rename the configuration